src/hooks/actions.tsx

```diff
@@ -209,8 +210,15 @@ export function useProposeAction({
       pending: <>Proposing price</>,
       success: <>Proposed price</>,
       error: <>Failed to propose price</>,
-    }).catch(console.error);
-  }, [proposePriceTransaction, chainId]);
+    })
+      .then((receipt) => {
+        if (receipt && query)
+          oracleEthersApis?.[query.oracleType]?.[
```

Comment:
this initiates the update by searching for the correct service api for the type and chain